### ISSUE TITLE
can't apply the extensions on files they are installing successfully can't able to use them applied some of the techniques like debug->Reload the extensions F5, and then uninstalled it and reinstalled it agin & again but can't make a impact!  

### ISSUE THREAD

**Issue by mohitbahrani** [adobe/brackets#13417]
### Prerequisites

* [ ] Can you reproduce the problem with `Debug -> Reload Without Extensions`?
* [ ] Did you perform a cursory search to see if your bug or enhancement is already reported?
* [ ] Did you read the [Troubleshooting guide](https://github.com/adobe/brackets/wiki/Troubleshooting)?

For more information on how to write a good bug report read [here](https://github.com/adobe/brackets/wiki/How-to-Report-an-Issue)
For more information on how to contribute read [here](https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md)

### Description

[Description of the bug or feature]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expected to happen]

**Actual behavior:** [What actually happened]

### Versions

Please include the OS and what version of the OS you're running.
Please include the version of Brackets. You can find it under `Help -> About Brackets` (Windows and Linux) or `Brackets -> About Brackets` (macOS)

**Comment by petetnt** [comment]
Hi @mohitbahrani,

could you point out some specific extensions / issues you are having?

**Comment by mohitbahrani** [comment]
Hi @petetnt,

No  it's happening with all the  extensions and ya  i also tried deleting it's registry  after uninstalling the brackets and after that reinstalling it but that too can't load the extensions in it please if you have some solutions for solving this error please do reply me

**Comment by petetnt** [comment]
Could you check Debug -> Show Developer Tools -> Console to see if there are any extension related errors?

**Comment by mohitbahrani** [comment]
@petetnt ,
yes, I checked that there are two errors  which are related to my 'C:' drive directory so i think there is some what issues with my windows only bcoz it has happened since i updated my windows
